README.md

```diff
@@ -1,5 +1,7 @@
 # Microsoft Azure SDK for .NET
- ----------
+
+[![Build Status](https://travis-ci.org/Azure/azure-sdk-for-net.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-net) [![Build Status](https://dev.azure.com/azure-sdk/public/_apis/build/status/net/azure-sdk-for-net%20-%20client?branchName=master)](https://dev.azure.com/azure-sdk/public/_build/latest?definitionId=41&branchName=master)
```

Comment:
Can you split this one up and label them as management and client? Similar to the java repo https://github.com/Azure/azure-sdk-for-java/pull/2605/files.

Reply:
Also while looking at the Java readme I noticed that the name of the build definition is part of the URL which we just changed so it is pointing at some old unknown state. According to the devops docs (https://docs.microsoft.com/en-us/rest/api/azure/devops/build/status/get?view=azure-devops-rest-5.0) and a quick test it appears that we can instead use definition id so can you please update this to use the id instead, as well as the other repos so we can prevent ourselves from breaking these if we change the name.

Reply:
For consistency between the status badge and the link we should also include the branch name. So for this one I'd have `https://dev.azure.com/azure-sdk/internal/_apis/build/status/41?branchName=master`

Reply:
I just noticed this one did have the branch already but the java one didn't. We should try to make them all 4 consistent and have them use the definition id and branch name.